src/ColumnSortable/Sortable.php

```diff
@@ -175,7 +175,13 @@ public static function link(array $parameters) //Extending Blade; Blade sends ar
             $anchorClass = 'class="' . $anchorClass . '"';
         }
 
-        return '<a ' . $anchorClass . ' href="'. url(Request::path() . '?' . $queryString) . '"' . '>' . htmlentities($title) . '</a>' . ' ' . '<i class="' . $icon . '"></i>';
+        $clickableIcon = Config::get('columnsortable.clickable_icon', false);
+        $trailingTag = ' ' . '<i class="' . $icon . '"></i>' . '</a>' ;
+        if ($linkOnIcon == false) {
```

Comment:
What is this $linkOnIcon variable; I mean it sure throws error `Undefined variable: linkOnIcon`.